### PR TITLE
fixed minor typos

### PR DESCRIPTION
**Pull request by tanTarantino**
### **Describe your change:**

- [ ] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new Java files are placed inside an existing directory.
- [ ] All filenames are in all uppercase characters with no spaces or dashes.
- [ ] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by tanTarantino** [comment]
Fixed some minor typos in Comments section of methods.

File: src/main/java/com/thealgorithms/backtracking/FloodFill.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 public class FloodFill {
 
     /**
-     * Get the color at the given co-odrinates of a 2D image
+     * Get the color at the given co-ordinates of a 2D image
```
Comment (siriak) [inline src/main/java/com/thealgorithms/backtracking/FloodFill.java:10]:
```suggestion
     * Get the color at the given coordinates of a 2D image
```

File: src/main/java/com/thealgorithms/backtracking/FloodFill.java [hunk 2]
```diff
@@ -19,9 +19,9 @@ public static int getPixel(int[][] image, int x, int y) {
     }
 
     /**
-     * Put the color at the given co-odrinates of a 2D image
+     * Put the color at the given co-ordinates of a 2D image
```
Comment (siriak) [inline src/main/java/com/thealgorithms/backtracking/FloodFill.java:22]:
```suggestion
     * Put the color at the given coordinates of a 2D image
```

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**